Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

```diff
@@ -26,16 +26,51 @@ When an instance of Umbraco starts up it generates some 'temporary' files on dis
 
 ### Umbraco PublishedCache
 
-When Azure Web Apps auto transitions between hosts, you scale the instances or you utilise slot swapping you may experience issues with the Umbraco Published Cache becoming locked unless Umbraco is configured to use SQL for MainDom locking. 
+When Azure Web Apps auto transitions between hosts, you scale the instances or you utilise slot swapping you may experience issues with the Umbraco Published Cache becoming locked unless Umbraco is configured correctly.
+
+#### Backoffice
+
+For the backoffice Web App **only** you should configure Umbraco to use SQL for MainDom locking. 
+
+Either in web.config or via the Azure Portal WebApp Application Settings section
 
 ```xml
 <add key="Umbraco.Core.MainDom.Lock" value="SqlMainDomLock" />
 ```
 
-:::note
-The `Umbraco.Core.MainDom.Lock` setting is for Umbraco v8.6+. Having this setting for versions between v8.0-v8.5 will not have any affect. It is recommended to use 8.6+ when running Umbraco on Azure Web Apps since this setting will prevent file locking issues.
-:::
+#### Front end
+
+ For the front end WebApp **only** instances you should configure disable overlapped recycling. 
```

Comment:
Configure '(Disable Overlapped recycling)[https://github.com/projectkudu/kudu/wiki/Configurable-settings#disable-overlapped-recycling]'

This setting **Only** applies to 'front end' replica WebApps - do not apply this to the WebApp used for backoffice editing.

Although this is an Application Setting, you cannot set this in you Web.Config file, it is not supported - you must add it to the Application Settings area of your Azure Portal.

Key: WEBSITE_DISABLE_OVERLAPPED_RECYCLING 
Value: 1

(This will turn off the ability for the WebApp to overlap during failover, and prevent locking issues from occurring)